File: Estatistica I/distribuicao_t.r
#distribuicao t-student

#media = 75, amostra = 9, dp = 10
# < 80, t = 1.5

pt(1.5, 8)

pt(1.5, 8, lower.tail = FALSE)

pt(1.5, 8) + pt(1.5, 8, lower.tail = FALSE)

1 - pt(1.5, 8)

File: Estatistica I/distribuicao_t.py
#ditribuicao t-student

from scipy.stats import t
#media = 75, amostra = 9, dp = 10
# < 80, t = 1.5

t.cdf(1.5, 8)

t.sf(1.5, 8)

t.cdf(1.5, 8) + t.sf(1.5, 8)

1 - t.cdf(1.5, 8)